docker/scripts/init

```diff
@@ -6,6 +6,11 @@ cd /app
 # Default to the environment variable values set in .env.example
 source /app/.env.example
 
+# Cleanup any leftover pid file
+if [ -f /app/tmp/pids/server.pid ]; then
+  rm -f /app/tmp/pids/server.pid
```

Comment:
Probably don't need the `-f` right?  Since you already checked for existence?  LGTM, merge when you're happy with it.